Modules/signalmodule.c

```diff
@@ -1250,6 +1253,38 @@ signal_pthread_kill_impl(PyObject *module, unsigned long thread_id,
 #endif   /* #if defined(HAVE_PTHREAD_KILL) */
 
 
+#if defined(__linux__) && defined(__NR_pidfd_send_signal)
+/*[clinic input]
+signal.pidfd_send_signal
+
+    pidfd: int
+    signalnum: int
+    siginfo: object = None
```

Comment:
IMHO, we already have a `SiginfoType` for siginfo.
Is there any reason not to check the type for it?

Reply:
Right now this parameter is required to be `None`.

Reply:
@benjaminp Got it :)